### PR TITLE
Added Lazyvim installation instructions

### PR DESCRIPTION
**Pull request by aindriu80**
Added lazyvim installation instructions - tested and working.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,56 @@ use {
 
 If you use another plugin manager just make sure `tabout.nvim` is loaded after `nvim-treesitter` and any completion that already uses your _tabkey_.
 
+### [lazyvim](https://www.lazyvim.org/)
```
Comment (abecodes) [inline README.md:68]: @aindriu80 Can you move your changes above line 66 please?

**Review by abecodes — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aindriu80) [reply README.md:65]: I've adjusted the location - please check.

Comment (abecodes) [reply README.md:65]: Awesome, thank you

**Review by abecodes — APPROVED** [review]

**Comment by abecodes** [comment]
@aindriu80 Thanks for your contribution ❤️

**Pull request merged**